dags.yaml

```
@@ -786,3 +786,23 @@ bqetl_newtab:
   schedule_interval: daily
   tags:
     - impact/tier_1
+
+bqetl_desktop_mobile_search_aggregates_monthly:
+  default_args:
+    depends_on_past: false
+    email:
+    - telemetry-alerts@mozilla.com
+    - akommasani@mozilla.com
+    email_on_failure: true
+    email_on_retry: true
+    end_date: null
+    owner: akommasani@mozilla.com
+    retries: 2
+    retry_delay: 30m
+    start_date: '2019-01-01'
+  description: Populate monthly aggregates based on daily clients table for desktop
+  schedule_interval: monthly
```

I believe this translates to 0 0 1 * * for 2nd of the month, seems like this should be:
```suggestion
  schedule_interval: 0 0 2 * *
```

---

I should have updated it here as well. But in the .py folder I set it to something like `schedule_interval="0 5 2 * *"` 5 am to allow time for upstream to run